src/WorkBC.Tests/Tests/SearchTests.cs

```diff
@@ -327,18 +327,18 @@ private JobSearchFilters GetFiltersForJobNocField(string nocCode)
             return filters;
         }
 
-        private JobSearchFilters GetFiltersForJobNocField2021(string nocCode2021)
-        {
-            var filters = new JobSearchFilters
-            {
-                Page = 1,
-                PageSize = 20,
-                SearchLocations = new List<LocationField>(),
-                SearchNoc2021Field = nocCode2021
-            };
-
-            return filters;
-        }
+        //private JobSearchFilters GetFiltersForJobNocField2021(string nocCode2021)
```

Comment:
You can delete this block completely, we don't want any record of `SearchNoc2021Field`.